### PR TITLE
skew second

### PR DESCRIPTION
**Pull request by przemas75**
This is something to solve https://github.com/aboe76/puppet-chrony/issues/19
Right now only for Debian and trying do so only for stretch and higher since i can see it is not implemented on jessie (chronyd not starting with these options set).

### CODE REVIEW

File: spec/classes/chrony_spec.rb [hunk 1]
```diff
@@ -125,6 +125,9 @@
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmdallow 1\.2\.3\.4$}) }
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmddeny 1\.2\.3$}) }
               it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*cmdallow all 1\.2$}) }
+              it { is_expected.to contain_file('/etc/chrony/chrony.conf').with_content(%r{^\s*leapsecmode slew$}) }
```
Comment (aboe76) [inline spec/classes/chrony_spec.rb]: You need to add these params to the code block above:
```
context 'with some params passed in' do
let(:params) do
{
            queryhosts: ['192.168/16'],
            port: '123',
            cmdport: '257',	
            config_keys_mode: '0123',
            config_keys_owner: 'steve',
            config_keys_group: 'mrt',
            config_keys_manage: true,
            chrony_password: 'sunny',
            bindcmdaddress: ['10.0.0.1'],
            leapsecmode: 'slew',
            maxslewrate: '1000',
            smoothtime: '400 0.001',
}
end
```

File: spec/classes/chrony_spec.rb [hunk 2]
```diff
@@ -130,6 +133,11 @@
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_group('mrt') }
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_replace(true) }
               it { is_expected.to contain_file('/etc/chrony/chrony.keys').with_content("0 sunny\n") }
+              if facts[:os]['distro']['release']['major'] > '8'
```
Comment (aboe76) [inline spec/classes/chrony_spec.rb]: why test for rhel 8 ?

File: spec/classes/chrony_spec.rb (hunk 1 above)
Comment (przemas75) [reply spec/classes/chrony_spec.rb]: yes, missed that. Thanks.

File: spec/classes/chrony_spec.rb (hunk 2 above)
Comment (przemas75) [reply spec/classes/chrony_spec.rb]: Well. That was funny.
I noticed on travis it fails, tried this and that.
Today, i cloned repo to tag 0.3.1 made modifications and it was all ok.
So i came to conclusion, 5 days ago i cloned repo with not all things ready (saw you updated a few things) and was modifying not ready code.

**Comment by aboe76** [comment]
@przemas75 nice work thanks for this, will upload a new release to the forge this weekend.

**Pull request merged**